implementation/src/main/java/io/smallrye/mutiny/Multi.java

```diff
@@ -193,9 +193,21 @@ static MultiCreateBy createBy() {
      *
      * @param executor the executor to use, must not be {@code null}
      * @return a new {@link Multi}
+     * @deprecated Use {@link #runSubscriptionOn(Executor)} instead.
      */
+    @Deprecated
     Multi<T> subscribeOn(Executor executor);
```

Comment:
Is it worth converting this and the one in `Uni` into default interface methods that call `runSubscriptionOn`?

Reply:
Good idea. Done.